### ISSUE TITLE
Access

### ISSUE THREAD

**Issue by kallewesterling** [DHRI-Curriculum/twitter-api#3]
From one of the DHRI participants:

Apologies if someone has already noted this, but one of our DHRI teachers has discovered that Twitter has, since last summer, restricted access to their API. In short, that means getting keys, access tokens, apps, etc, requires a special Developer account. HOWEVER, fear not. According to a process for academics outlined here, you as teacher can start an organizational account and then invite students to join, which will give those students the power to create an app. 

I recently applied for this organization account status, and after a day and half, was approved. It's a bit of a pain to fill out the forms, but not terrible. We have not yet tested out how this works on the ground, but we'll need to do at least a small update to the forked version of the curriculum (and hopefully not a major revision). I'm a little curious whether anyone else has encountered this.

Anyway, just a quick heads up for those planning to teach the Twitter module. If this is news out of the blue to you, as it was to me, it's because those accounts that already have developer access were "grandfathered in," and so you wouldn't have noticed anything, which would have been a nasty surprise when new Twitter users in your class couldn't get access to the Twitter app dashboard. (Granted, I may have missed a Twitter email along the way, so maybe this isn't news to you). And, if I'm totally misunderstanding things here, I'd be very happy to hear that I'm mistaken.

**Comment by smythp** [comment]
Steve and I have this additional access now, but I haven't tested it extensively. In theory it shouldn't be a problem, but I'd be curious to know how fast students can be onboarded with this new process. It does seem likely te tutorial will need to be updated.

**Comment by kallewesterling** [comment]
Yes, definitely. I'm going to look it over and make sure that we make necessary edits. The comment above is from one of the DHRI participants who ran into trouble trying to go through the workshop before their institute..!
